coc.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Code of Conduct
+categories: nav
+showinnav: true
+---
+
+# {{ page.title }}
```

Comment:
nit: remove `# {{ page.title }}`

Reply:
Every page has page.title at the top. Rules is the worst offender. Minor detail we won't worry about in this PR.